src/analyses/base.ml

```diff
@@ -2131,7 +2131,8 @@ struct
           ignore (Priv.enter_multithreaded (Analyses.ask_of_ctx ctx) (priv_getg ctx.global) (priv_sideg ctx.sideg) st);
         Priv.threadenter (Analyses.ask_of_ctx ctx) st
       ) else
-        let globals = CPA.filter (fun k v -> Basetype.Variables.is_global k) st.cpa in
```

Comment:
I think we should remove `Basetype.Variables.is_global` altogether, it seems to be only used in a few places and can clearly be misleading since it's only `x.vglob` check. The few remaining uses should either use the ask version or explicitly check the field if they for some reason shouldn't account for escapes.

Reply:
Removed it now!